sample_size/sample_size_calculator.py

```diff
@@ -0,0 +1,106 @@
+from typing import Any
+from typing import List
+
+import numpy as np
+from statsmodels.stats.power import NormalIndPower
+from statsmodels.stats.power import TTestIndPower
+
+from sample_size.variables import Boolean
+from sample_size.variables import Numeric
+from sample_size.variables import Ratio
+
+DEFAULT_ALPHA = 0.05
+DEFAULT_POWER = 0.8
+
+
+class PowerAnalysisParameters:
```

Comment:
I notice we use this class in the following way:

```python3
metric = PowerAnalysisParameters(variable.variance, mde, self.alpha, self.power)
```

I wonder if it might make more sense to call this class `Metric`? Jumping off of that, perhaps a metric would simply have a `Variable` (`Boolean`, `Ratio`, etc.) and an MDE? `alpha` and `power` seem to already be handled by the `SampleSizeCalculator` class?

Reply:
Yeah that's what I plan to do if no objections.